### ISSUE TITLE
new website: home/index page goals

### ISSUE THREAD

**Issue by Anton-4** [roc-lang/roc#5740]
- [ ] should guide you to the information you need:
    + install roc 
    + tutorial 
    + examples
    + community page
    + how to sponsor roc
    + docs page
    + zulip chat
    + foundation website (foundation.roc-lang.org/)
- [ ] state the goals of roc

- [ ] mention the pre-release, work-in-progress state of roc.
- [ ] briefly mention the open, permissive license
- [ ] link to the roc twitter and github page
- [ ] thank sponsors
- [ ] should follow the general goals in https://github.com/roc-lang/roc/issues/5739

stretch goals:
- [ ] Integrate the web repl, currently at https://www.roc-lang.org/repl. #5136 and #5301 should be done first.

Good examples:
- https://www.rust-lang.org/

Where's the code?
https://github.com/roc-lang/roc/blob/main/www/wip_new_website/content/index.md
https://github.com/roc-lang/roc/blob/main/www/wip_new_website/static/site.css
